pkg/flavor/kubernetes/flavor.go

```diff
@@ -411,45 +411,13 @@ func (flavor *Flavor) getClaimOverrideOptions(claim *v1.PersistentVolumeClaim, o
 		}
 	}
 
-	// check to see if there is cloneOfPVC annotation present
-	volumeName, err := flavor.getPVFromPVCAnnotation(claim, provisioner)
-	if err != nil {
-		return nil, err
-	}
-
-	// update the options map with the pv name if there is one
-	if volumeName != "" {
-		optionsMap[cloneOf] = volumeName
-	}
-
 	// make sure cloneOfPVC is removed from the options (all cases)
 	if _, found := optionsMap[cloneOfPVC]; found {
 		delete(optionsMap, cloneOfPVC)
 	}
```

Comment:
Why don't we drop this too?  Along with the constant?

Reply:
I just kept it in case if anyone trying to use this option  from flexvolume references. CSP will ignore it in that case?

Reply:
But this is the CSI code base.  Support for this is implemented via DataSource.

Reply:
sure, i just tested by passing cloneOfPVC without removing from option map. will cleanup this too.

Reply:
done..